### ISSUE TITLE
Registry server for tests

### ISSUE THREAD

**Issue by joehand** [datproject/dat#682]
Right now we are using the datproject/datproject.org repo for testing registry commands. This is not ideal because we get some test failures from datproject.org changes, e.g. https://travis-ci.org/datproject/dat/jobs/210658187. 

We need some way to test auth/publish commands, maybe we don't need to test against the actual server? Or we can move the server to a separate module?

**Comment by joehand** [comment]
https://github.com/datproject/datproject.org/issues/458

**Comment by karissa** [comment]
yeah, moving the api server to a separate module is ideal! https://github.com/datproject/datproject.org/issues/458
